deployment/submit_emr_job_train.py

```diff
@@ -139,7 +142,10 @@ def submit_job(input_bootstrap_file, input_src_code_file):
                 'ActionOnFailure': 'TERMINATE_CLUSTER',
                 'HadoopJarStep': {
                     'Jar': 'command-runner.jar',
-                    'Args': ['python3.6', '/home/hadoop/CVAE/test_cvae-packagedata5.py']
+                    'Args': ['/usr/local/bin/python3.7'.format(
```

Comment:
`>>> '/usr/local/bin/python3.7'.format( '/home/hadoop/fabric8-analytics-npm-insights')
'/usr/local/bin/python3.7'`

This is what this evaluates to, so either add a formatting variable to remove the `.format()`

Reply:
Yeah, you are correct here, but I picked this logic from Rudra lib. https://github.com/fabric8-analytics/fabric8-analytics-rudra/blob/master/rudra/deployments/emr_scripts/emr_config.py#L63.